src/cryolike/metadata/lens_descriptor.py

```diff
@@ -19,6 +19,8 @@ class RelionField(NamedTuple):
     RelionField('DefocusU', 'defocusU', '', None, False, True),
     RelionField('DefocusV', 'defocusV', '', None, False, True),
     RelionField('DefocusAngle', 'defocusAngle', '', None, False, True),
+    RelionField('OriginXAngst', 'originXAngst', 'x shift from origin', 0, True, False),
+    RelionField('OriginYAngst', 'originYAngst', 'y shift from origin', 0, True, False),
```

Comment:
I didn't catch this until I was looking at the tests below--subtle!

My (undocumented) way of thinking was that a field that is defaultable should always be marked as "required", because it will never be null in the fully constructed object--if it's missing, the default will be used. So defaultable implies required.

Other than the test you had to change, I'm not sure if this really impacts anything, but would you be comfortable humoring me on this one?